src/main/java/sqltoregex/deparser/ExpressionDeParserForRegEx.java

```diff
@@ -531,10 +535,13 @@ public void visit(Function function) {
                     StringBuilder tableNameWithAlias = new StringBuilder();
                     if(singleExpression.contains(".")){
                         String tableName = singleExpression.split("\\.")[0];
+
+                        tableNameWithAlias.append("(");
                         deparseTableName(tableName, tableNameWithAlias);
                         if (!tableName.isEmpty()) {
                             tableNameWithAlias.append(".");
                         }
+                        tableNameWithAlias.append(")?");
```

Comment:
May be okay for "kompetenzorientiertes prüfen"

But collides with in INNER JOIN Scenarios with ambiguous column names:

SELECT col1 FROM table1 INNER JOIN table2 WHERE  col1=col1

I think this is something Mr. Rohland wouldn't allow.

Reply:
see #117